climate_scanner/coarse_article_classifier/coarse_classifier.py

```diff
@@ -1,66 +1,50 @@
-            }
-
-        return data
-
-    def pre_processing(self, data):
-        """
-        function to clean the data, numerify the data without losing too much information (feature engineering)
-        """  
-        final__data = {
-                "ID" : 1545,
-                "title" : "The Shattering Truth of 3D-Printed Clothing",
-                "cleaned_doc" : "  "
-            }
-
-        return final__data
-
-    def fit(self, final__data):
+    def train(self, final__data):
         """
         1. split data to training and test set
         2. fit the model on training data
         3. parameter tuning
         """
-        x_train, y_train, x_test, y_test = # training and test set
-
-        # create an object for the model being used. For the sake of understanding, I'm creating an object for Logistic regression here.
-
-        self.model = LogisticRegression()
-        self.model.fit(x_train, y_train)
+        pass
 
-        self.x_test = x_test
-        self.y_test = y_test
+    def test(self):
+        """
+        A function to test the classifier.
+        :return:
+        """
+        pass
     
     def predict(self, test__data):
         """
         predict classes (relevant/not relevant) for the test data
         """
-        y_pred = self.model.predict(self.x_test)
-
-        predictions = {
-                "ID" : 1545,
-                "title" : "The Shattering Truth of 3D-Printed Clothing",
-                "climate_scanner" : True,
-                "climate_scanner_prob" : 0.9
-            }
-
-        return predictions
+        pass
     
     def evaluate(self, predictions):
         """
         evaluate the predictions
         """
+        pass
+
+class Doc2Climate:
+
+    def __init__(self):
+        self.climate_classifier = ClimateClassifier()
+
+    def coordinator_pipeline(self,input_dict=None):        
+        text = input_dict["text"]
+        all_predictions = []
+        for dic in text:
+            processed_data = DataUtils.pre_processing(dic["text"])
```

Comment:
Looks great. One change though - please make it so that these functions return some kind of fake output. I want to be able to send the pipeline an input and get a fake result back :).

Reply:
Added